src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -0,0 +1,612 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Silk.NET.GLFW;
+using Silk.NET.Windowing.Common;
+using Ultz.Dispatcher;
+
+namespace Silk.NET.Windowing.Desktop
```

Comment:
You also need to make a class that implements `ISilkPlatform` called `GlfwPlatform`.

Reply:
Actually, the whole `ISilkPlatform`/`Silk.Init` stack needs to be implemented in Windowing.Common too. Just follow the proposal for that, any implementations you want me to do (i.e. the Silk.Init method) just throw a `NotImplementedException`

Reply:
Got the skeleton done, but IsApplicable and Silk.Init falls to you since I have no idea how to do that